HIP/hip-868.md

```diff
@@ -0,0 +1,167 @@
+---
+hip: 868
+title: Support Cancun Self-Destruct Semantics in Smart Contract Services
+author: Danno Ferrin (@shemnon)
+working-group: Nana Essilfie-Conduah <@nana-ec>, Jasper Potts <@jasperpotts>, Richard Bair <@rbair23>, Stoyan Panayotov <stoyan.panayotov@limechain.tech>
+type: Standards Track
```

Comment:
We added a new header for council members to see who requested hips. Could you add the 'requested-by' header here and add whoever requested this hip?

Reply:
This is part of EVM equivalence and ensuring the network continues to support the latest Ethereum Hardfork features.
There's no one or few requestors. Just the network staying up to date to ensure support